### ISSUE TITLE
wrong ID (possibly just a text error)

### ISSUE THREAD

**Issue by RandomIntermition** [Steveice10/FBI#31]
after installing FBI.cia over H&S. (1.4.17)
on screen text claims it is "fbi 1.3.8"
double checked and i'm definitely working with 1.4.17

**Comment by Steveice10** [comment]
If it says 1.3.8, you're on an old version. The newest version correctly displays "1.4.17".

**Comment by RandomIntermition** [comment]
i downloaded it direct from the git release though :S (and i wasn't working from a 'starter' folder, so no chance of it including any older versions)

**Comment by Steveice10** [comment]
How did you inject it, though? I'm fairly certain at least rxTools's FBI injection uses 1.3.8.

**Comment by RandomIntermition** [comment]
hey, i injected using rxtools injector. i just checked and it hasn't been updated in a while.
BUT... form my understanding the injector doesn't include FBI. it just facilitates the ability to inject official FBI into H&S, which you need the injector and FBI on SD for it to do.

am i wrong?
i tried injecting without putting 1.4.17 on SD and it just said can't find fbi files, so assumed it was installing from them.
i thought maybe the fact it was outdated compared to FBI meant a text error claiming it to be an older version when it wasn't.(seen as i've used it to install relatively new games with no problems.

**Comment by RandomIntermition** [comment]
actually, i've just been checking features etc. you're right it is 1.3.8, doesn't include anything added after that.

as a work around until rxtools is updated (as i don't have the tool/code used to create the injector...
i've installed 1.3.8 with injector then used that to install 1.4.17 CIA.

can you just clarify for me that installing the current CIA from within the 1.3.8 CIA won't cause any problems/conflicts/errors when it comes to using 1.4.17 as my main cia installer.
obviously older/newer software can clash but as far as i'm aware injector(using H&S) would have to be installed in nand where as the CIA can be installed to SD. so shouldn't be any problems.

in short, just unsure if errors in 1.3.8 might cause 1.4.17 to not be installed properly if installing one from the other.

(sorry for the long comments)

if you can just clarify that for future ref, thanks.

**Comment by Steveice10** [comment]
You shouldn't have any issues with 1.4.17 being installed normally as well as 1.3.8 being injected into H&S.
